change/@ni-nimble-components-c9f2f270-d0d3-478a-b43a-c0777af5fc44.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Friendly reminder to approve the Chromatic checks before submitting.

Reply:
Sure, I can do that. I've gotten mixed feedback on whether I should be doing that or whether there should be a reviewer that acts as a second set of eyes to make sure we're happy with the detected Chromatic changes.

Reply:
Fair point. We should probably revisit with the team what we want our policy to be. I don't think we really follow what we documented. I'll try to remember to bring it up at standup.